web/src/routes/(user)/map/+page.server.ts

```diff
@@ -0,0 +1,23 @@
+import { AppRoute } from '$lib/constants';
+import { redirect } from '@sveltejs/kit';
+import type { PageServerLoad } from './$types';
+
+export const load = (async ({ locals: { api, user } }) => {
+	if (!user) {
+		throw redirect(302, AppRoute.AUTH_LOGIN);
+	}
+
+	try {
+		const { data: mapMarkers } = await api.assetApi.getMapMarkers();
```

Comment:
Might be worth to only load the markers client-side to improve the initial page load time

Reply:
I experimented with that. My main reason was the following: The initial centering of the map depends on the loaded data. Loading client-side results in a change of the center point very quickly after opening the map tab.
This doesn't look very nice and also results in a lot of additional tile loads.

I found a little delay but complete map from the beginning a bit cleaner looking.

Reply:
@michelheusschen After sleeping a night, I think I'll try some hybrid implementation, you made ideed a valid point here. Maybe I extend the API endpoint so I can query the first, lets say 1000 assets server side and the rest client side.